nautobot/ipam/tests/test_filters.py

```diff
@@ -837,12 +834,14 @@ def setUpTestData(cls):
         )
 
     def test_name(self):
-        params = {"name": VLAN.objects.all().values_list("name", flat=True)[:2]}
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
+        names = list(VLAN.objects.all().values_list("name", flat=True))[:2]
```

Comment:
I'm realizing in cases like this it's gonna be a tad more optimal to slice the queryset first, and cast to a list second, assuming we have more than a handful of VLAN objects, that is.

For example:

```suggestion
        names = list(VLAN.objects.all().values_list("name", flat=True)[:2])
```

Perhaps since we're starting to repeat this pattern a lot as we refactor to use the fixture factories, we might consider factoring it into a test case helper method? Just thinking aloud. Please don't change it right now.

Reply:
Good thought. Since we've already got this pattern in a bunch of tests after #2479 and #2514, maybe we should get a housekeeping item open to clean up all of these?

Reply:
Solid plan.